charts/tidb-lightning/templates/scripts/_start_lightning.sh.tpl

```diff
@@ -29,7 +29,11 @@ fi
 {{- if eq .Values.backend "importer" }}
     --importer={{ .Values.targetTidbCluster.name }}-importer.{{ .Values.targetTidbCluster.namespace | default .Release.Namespace }}:8287 \
 {{- else if eq .Values.backend "tidb" }}
-    --backend tidb \
+    --backend=tidb \
+    --tidb-port=4000 \
```

Comment:
for `tidb` backend, we should set `tidb-port` as well because TiDB-Lightning wouldn't auto setting this based on `/settings`.

if we do not need `tidb` backend in the chart anymore, maybe we can just remove it?

Reply:
Sorry that I did not notice that we support tidb backend in the chart, is `tidb-port` the only parameter missed? Can we just let the user to set the `config` filed in the values.yaml and we have to make it clear in the doc.

Reply:
YES, currently only `tidb-port` missed and I have tested manually (https://github.com/pingcap/tidb-lightning/blob/463e7289fcbc412a49e50b307e3b33d30da7f218/lightning/config/config.go#L591-L614). I think we can add it in this script as we did for `2379` and `8287`. BTW, we also hardcode `tidb-host` in this script.